var/spack/repos/builtin/packages/nwchem/package.py

```diff
@@ -26,7 +28,8 @@ class Nwchem(Package):
     depends_on('mpi')
     depends_on('scalapack')
 
-    depends_on('python@2.7:2.8', type=('build', 'link', 'run'))
+#    depends_on('python@2.7:2.8', type=('build', 'link', 'run'))
+    depends_on('python@3:', type=('build', 'link', 'run'))
```

Comment:
Do older versions still need Python 2? If so, it should look like:
```python
depends_on('python@3:', when='@7:', type=('build', 'link', 'run'))
depends_on('python@2.7:2.8', when='@:6', type=('build', 'link', 'run'))
```

Reply:
Yes, older version need it. Can we obsolete some of the older versions? I would like to remove 6.8

Reply:
Commit https://github.com/spack/spack/pull/17779/commits/e28278744510c5fff0a37e4229ff393137406563 brings back python2 for 6.8.1